command/agent/consul/self.go

```diff
@@ -0,0 +1,56 @@
+package consul
+
+import (
+	"strings"
+
+	"github.com/hashicorp/go-version"
+)
+
+// Self represents the response body from Consul /v1/agent/self API endpoint.
+// Care must always be taken to do type checks when casting, as structure could
+// potentially change over time.
+type Self = map[string]map[string]interface{}
+
+func SKU(info Self) (string, bool) {
```

Comment:
When the `SKU` and `Namespaces` functions were in the fingerprinter, it made sense that they returned `(string, bool)`. But now that they're in a different package entirely, their signature is "weird". For `SKU` the bool is false iff the string is empty, and for `Namespaces` we're returning a stringified version of the bool.

Maybe it'd be better if `SKU` returned just a string and `Namespaces` returned just a bool, and let their callers in the fingerprint extractors and namespace client sort out returning the values the fingerprinter expects?

Reply:
Good idea, I'll do this

Reply:
Did this in 4b3ed5351